packages/cli/src/config/schema.ts

```diff
@@ -241,6 +241,20 @@ export const schema = {
 			env: 'EXECUTIONS_MODE',
 		},
 
+		concurrency: {
+			doc: 'Max number of executions to run in parallel in a single container/process',
+			format: Number,
+			default: 10,
+			env: 'EXECUTIONS_CONCURRENCY',
+		},
+
+		manualConcurrency: {
+			doc: 'Max number of manual executions to run in parallel in a single container/process',
+			format: Number,
+			default: 3,
```

Comment:
Too low? Should we minimize the chance for users to experience being rate limited?